plantswap/components/DetailpageHeaderImage/DetailpageHeaderImage.js

```diff
@@ -0,0 +1,46 @@
+"use client"
+
+import { useEffect, useRef, useState } from 'react';
+import NextImage from 'next/image';
+import ColorThief from 'colorthief';
+import styles from '../DetailpageHeaderImage/detailpageheaderimage.module.scss';
+
+const DetailpageHeaderImage = ({ data, onColorExtracted }) => {
```

Comment:
mabye call it detail banner? much shorter

Reply:
Or DetailPageHero / DetailPageHeader